smf/zone-network-setup/method_script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I would just not use a shell script for the service method (if you do, it should be in `ksh`, not `bash` since it is run from SMF fairly early).

I would use property tokens in the start method in the manifest, that is, the start method would look something like the following, then you don't need this script at all.

```xml
  <exec_method type='method' name='start'
    exec='/opt/oxide/zone-network-setup/bin/zone-networking -d %{config/datalink} -l %{config/listen_addr} -g %{config/gateway}'
    timeout_seconds='0' />
```